packages/sku/src/context/index.ts

```diff
@@ -147,47 +147,45 @@ if (devServerMiddleware && !useDevServerMiddleware) {
 }
 
 export const paths = {
-  appSkuConfigPath,
-  devServerMiddleware,
-  src: skuConfig.srcPaths.map(getPathFromCwd),
-  /** @type {string[]} */
-  compilePackages: [...defaultCompilePackages, ...skuConfig.compilePackages],
-  clientEntry: getPathFromCwd(skuConfig.clientEntry),
-  renderEntry: getPathFromCwd(skuConfig.renderEntry),
+  appSkuConfigPath: appSkuConfigPath!,
+  devServerMiddleware: devServerMiddleware!,
+  src: skuConfig.srcPaths!.map(getPathFromCwd),
+  compilePackages: [...defaultCompilePackages, ...skuConfig.compilePackages!],
```

Comment:
I know that we need all these `!`s right now, but removing in the future is high priority as they're just _the worst_.

Reply:
Agreed. It was either this or do a `||` short circuit to ensure it had values.